src/main/java/picard/analysis/AlignmentSummaryMetricsCollector.java

```diff
@@ -270,7 +268,8 @@ private void collectQualityData(final SAMRecord record, final ReferenceSequence
                 // NB: for read count metrics, do not include supplementary records, but for base count metrics, do include supplementary records.
 
                 // If the read isn't an aligned PF read then look at the read for no-calls
-                if (record.getReadUnmappedFlag() || record.getReadFailsVendorQualityCheckFlag() || !doRefMetrics) {
+                if (!record.getSupplementaryAlignmentFlag() &&
```

Comment:
Why is this check added `getSupplementaryAlignmentFlag`?  I thought we _wanted_ to have supplementary alignments add to these metrics (ex. `PF_HQ_MEDIAN_MISMATCHES`)? @yfarjoun want to weigh in here?

Reply:
Done, changed if-else condition, sorry for this omission.